environment.yml

```diff
@@ -0,0 +1,24 @@
+channels:
+- conda-forge
+- anaconda
+- defaults
+dependencies:
+- constructor
+- gcc
+- make
+- freetype
+- libsm-cos6-x86_64
+- curl
+- wget
+- unzip
+- git
+- ffmpeg
+- pkg-config
+- nodejs
+- cudatoolkit=11.0 # Adjust the version number as needed
+- pip:
+  - pip
+  - nvidia-cudnn-cu11==8.6.0.163 # if GPU_FLAG==1
+  - nbformat
+  - jupyterlab
```

Comment:
@ctr26 is jupyter lab needed for colab? I'm a bit confused with the environment: would this one be used both for colab and the containerisation? for the latter, I think nvidia and cuda drivers/packages should not be included in the conda environment as they are already predefined with the docker images.

Reply:
Yep, I've mostly copied the contents of what's in Ivan's dockerfile so that we can eventually migrated the docket file to Conda for dependency management (it's in one of the issues). Installing julyterlab is fine on company and it's nice that the environment file will be the same for colab and jupyter

Reply:
ah! ok, yes for jupyter lab. I thought this could affect colab installation.

the one I'm concerned about is nvidia-cudnn as this one is different for different notebooks. @IvanHCenalmor  can you please confirm that this is always installed in the dockerfile no matter the notebook?

Reply:
Yes, the nvidia-cudnn library is always installed. Its in the Dockerfile: https://github.com/HenriquesLab/DL4MicEverywhere/blob/9cf9476e4632a6d5dc90eb55469e0a40a3abceb7/Dockerfile#L39
Maybe it needs to be different depending on the CUDA version, but until now with that line is has always worked.